### DISCUSSION TITLE
How to find docker image corresponding to a specific release?

### DISCUSSION THREAD

**Discussion by prakash-42** [provectus/kafka-ui#4366] [General]
While the release versions in this project follow semantic versioning (the last version was 0.7.1), I am **not able to find the docker image that corresponds to this release**. The docker image tags for this project in Docker Hub are some cryptic hashes that I can't understand: https://hub.docker.com/r/provectuslabs/kafka-ui/tags . 

I have some reservations against using the `latest` tag in production, as the underlying image will change with every release. Is there a mapping with the tags in docker hub with the actual releases in this repo? Please help me with picking which docker image to use.

**Comment by erlfos** [comment]
You can use "Filter tags" to filter for example everything containing 'v' (v0.7.1 for example):
![image](https://github.com/provectus/kafka-ui/assets/45815831/3490f893-e506-402c-bdc3-03a255c81100)
Not ideal, but it works.

**Reply by prakash-42** [reply]
Thanks, that's exactly what I was looking for!
